web/client/components/manager/users/GroupDialog.jsx

```diff
@@ -22,11 +22,13 @@ const Dialog = require('../../../components/misc/Dialog');
 const assign = require('object-assign');
 const Message = require('../../../components/I18N/Message');
 const Spinner = require('react-spinkit');
-const Select = require("react-select");
 const {findIndex} = require('lodash');
+const PagedCombobox = require('../../misc/combobox/PagedCombobox');
```

Comment:
I suggest you to create an enhancer for the PagedCombobox to separate functionalities of loading from the window.